Makefile

```diff
@@ -160,23 +160,29 @@ endif
 .PHONY: install
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/crd | kubectl apply -f -
-	kubectl apply -f config/daemon/namespace.yaml # Hack to get install to behave.
-	$(KUSTOMIZE) build config/daemon | kubectl apply -f -
 
 .PHONY: uninstall
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
-	$(KUSTOMIZE) build config/daemon | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
-	kubectl delete -f config/daemon/namespace.yaml --ignore-not-found=$(ignore-not-found)
+
+.PHONY: deploy-kind
+deploy-kind: manifests kustomize install-cert-manager ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
pls update the comment stating this deploy on KIND while the other deploy on OCP